app/views/volunteers/show.html.slim

```diff
@@ -0,0 +1,115 @@
+.row
+  .col-xs-12
+    h1
+      = @volunteer.first_name
+      | ,
+      = @volunteer.last_name
```

Comment:
Define a method, called full_name in the model and which returns `#{ last_name }, #{ first_name }` and then just call this here